src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -453,3 +452,31 @@ def get_measured_sbp(
             randomness=self.randomness,
             additional_key="measured_sbp",
         )
+
+    def get_ascvd(self, visitors: pd.DataFrame) -> pd.Series:
+        """Calculate the atherosclerotic cardiovascular disease score"""
+        df_visitors = self.population_view.get(visitors)
+        return (
+            -19.5
```

Comment:
Do you think it would be worth storing these as constants? It looks to me like they're just coefficients and an intercept so I can see the argument for keeping it as is.

Reply:
Yeah, it might be. I did store the coefficients for the baseline medication coverage. This felt a bit different b/c it's a bit more of a "known" equation (whereas the baseline coverage was based on some simulation someone did); it'd be kind of like storing pi instead of just saying C = np.pi * D.

Reply:
@hussain-jafari I moved the coefficients into data_values like you recommended